test/blackbox-tests/test-cases/pkg/oneshot-webserver-large-file-curl-error.t

```diff
@@ -0,0 +1,17 @@
+Reproduction for an issue with the oneshot webserver used for testing
+package management, where attempting to send a large file causes the
+connection to be reset before the transfer completes.
+
+Make a large file:
+  $ dd if=/dev/zero of=./large-file iflag=fullblock,count_bytes status=none count=20M
```

Comment:
I have tested on my machine, and it starts failing from 14M, yeah 20M could be better number.